src/docfx/config/Config.cs

```diff
@@ -181,5 +182,14 @@ internal sealed class Config
         /// for the latest commit that touches that document.
         /// </summary>
         public readonly bool UpdateTimeAsCommitBuildTime = false;
+
+        [OnDeserialized]
+        private void OnDeserialized(StreamingContext context)
```

Comment:
can we add [ToLowercase] attribute for that? then we can also use it for other property like locale.

Reply:
attributes are very restrictive, in this case, `BaseUrl` depend on the value of `Output.LowerCaseUrl` to determine whether it needs to lower.